supabase/migrations/21_billing_v2.sql

```diff
@@ -174,7 +177,13 @@ begin
 
   -- Verify that the user has an admin grant for the requested `billed_prefix`.
   perform 1 from auth_roles('admin') as r where billed_prefix ^@ r.role_prefix;
-  if not found then
+  has_admin_grant = found;
+
+  -- Check whether user has bypassrls flag
+  perform 1 from pg_roles where rolname = session_user and rolbypassrls = true;
+  has_bypassrls = found;
+
+  if not has_bypassrls and not found then
```

Comment:
`if not has_bypassrls and not has_admin_grant then`

Reply:
🤦 thanks